src/Shared/SFA.DAS.ProviderPayments.Calc.Shared.Infrastructure/Data/Entities/RefundPaymentEntity.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace SFA.DAS.ProviderPayments.Calc.Shared.Infrastructure.Data.Entities
```

Comment:
I don't think that we need this. There is no difference between one of these and a paymentsentity

Reply:
OK, will be removed